### PR TITLE
Kotlinx Coroutines 1.5.0

### PR DESCRIPTION
**Pull request by 4brunu**
Kotlinx Coroutines version bump

### CODE REVIEW

**Comment by 4brunu** [comment]
@moljac since you were upgrading KotlinX Coroutines and the version 1.5.0 was released on Friday, I opened this PR to try to help with the KotlinX Couroutines upgrade

File: Android/KotlinXCoroutines/samples/KotlinxCoroutinesSample/KotlinxCoroutinesSample.csproj [hunk 1]
```diff
@@ -104,7 +104,7 @@
     <PackageReference Include="Xamarin.Google.Android.Material" Version="1.3.0.1" />
     <PackageReference Include="Xamarin.AndroidX.Legacy.Support.Core.Utils" Version="1.0.0.7" />
   <PackageReference Include="Xamarin.AndroidX.Migration" Version="1.0.8" />  <PackageReference Include="Xamarin.Kotlin.StdLib">
-  <Version>1.4.32.1</Version>
+  <Version>1.5.0.1</Version>
```
Comment (moljac) [inline Android/KotlinXCoroutines/samples/KotlinxCoroutinesSample/KotlinxCoroutinesSample.csproj:107]: ~~no need for .1, just 1.5.0~~

Comment (moljac) [reply Android/KotlinXCoroutines/samples/KotlinxCoroutinesSample/KotlinxCoroutinesSample.csproj:107]: ups. sorry. Didn't see it is Kotlin.StdLib

**Review by moljac — APPROVED** [review]

**Comment by moljac** [comment]
LGTM

File: Android/KotlinXCoroutines/samples/KotlinxCoroutinesSample/KotlinxCoroutinesSample.csproj (hunk 1 above)
Comment (4brunu) [reply Android/KotlinXCoroutines/samples/KotlinxCoroutinesSample/KotlinxCoroutinesSample.csproj:107]: 👍

**Comment by 4brunu** [comment]
@moljac this is the current error, but I don't know how to fix them, or what's the right thing to do here. 
Do you have any ideia?
```
obj/Release/monoandroid9.0/generated/src/Xamarin.KotlinX.Coroutines.DelayKt.cs(72,49): error CS0111: Type 'DelayKt' already defines a member called 'Delay' with the same parameter types [XamarinComponents/Android/KotlinXCoroutines/source/KotlinX.Coroutines.Core.Jvm/KotlinX.Coroutines.Core.Jvm.csproj]
obj/Release/monoandroid9.0/generated/src/Xamarin.KotlinX.Coroutines.TimeoutKt.cs(61,49): error CS0111: Type 'TimeoutKt' already defines a member called 'WithTimeout' with the same parameter types [XamarinComponents/Android/KotlinXCoroutines/source/KotlinX.Coroutines.Core.Jvm/KotlinX.Coroutines.Core.Jvm.csproj]
obj/Release/monoandroid9.0/generated/src/Xamarin.KotlinX.Coroutines.TimeoutKt.cs(99,49): error CS0111: Type 'TimeoutKt' already defines a member called 'WithTimeoutOrNull' with the same parameter types [XamarinComponents/Android/KotlinXCoroutines/source/KotlinX.Coroutines.Core.Jvm/KotlinX.Coroutines.Core.Jvm.csproj]
obj/Release/monoandroid9.0/generated/src/Xamarin.KotlinX.Coroutines.Flow.FlowKt.cs(928,70): error CS0111: Type 'FlowKt' already defines a member called 'Debounce' with the same parameter types [XamarinComponents/Android/KotlinXCoroutines/source/KotlinX.Coroutines.Core.Jvm/KotlinX.Coroutines.Core.Jvm.csproj]
obj/Release/monoandroid9.0/generated/src/Xamarin.KotlinX.Coroutines.Flow.FlowKt.cs(2151,70): error CS0111: Type 'FlowKt' already defines a member called 'Sample' with the same parameter types [XamarinComponents/Android/KotlinXCoroutines/source/KotlinX.Coroutines.Core.Jvm/KotlinX.Coroutines.Core.Jvm.csproj]
```

**Comment by moljac** [comment]
@4brunu Bom dia and thanks for reporting.

I'll need to take a look... it can be quite a lot of things involved. Generic parameters as `Java.Lang.Object` and then you have 2 method/function signatures that are the same in managed world, but not in java world... 

Right now I need to publish tons of new AndroidX packages...

**Comment by 4brunu** [comment]
Nice touch with bom dia :)
Dobro jutro! (I think? 🤔 )
Go ahead, and thanks for helping 👍

**Comment by 4brunu** [comment]
@moljac CI is now passing.
The solution was to remove some experimental API.

**Review by moljac — APPROVED** [review]

**Pull request merged**